### DISCUSSION TITLE
CropByLabelClassesd

### DISCUSSION THREAD

**Discussion by oscarramirezs** [Project-MONAI/tutorials#994] [Q&A]
Hi, 
Im working in a projet and I need to cut the ct images in the same shape that the label, 
that means all 0 in the label became 

I am working on a project and I need to crop the ct images in the same way as the label, that means all 0' in the label became 0 in the CT as well;
I'm using RandCropByLabelClassesd

        RandCropByLabelClassesd(keys=["image"], label_key="label", spatial_size=None, ratios=None, 
                                num_classes=3, num_samples=1, ),
 there is a way to do this with monai.transform?

![Capture](https://user-images.githubusercontent.com/52659349/196231741-a4568e2b-efdb-47e8-ae46-be707451b2f9.JPG)

someone can help me 

sorry but im new on this 

Oscar

**Comment by wyli** [comment]
I guess you can include `label` in the `keys`:
```py
RandCropByLabelClassesd(keys=["image", "label"], label_key="label", spatial_size=None, ratios=None, 
                            num_classes=3, num_samples=1, ),
```

**Reply by oscarramirezs** [reply]
I did it, but the ct still the same. 

thanks 
Oscar

**Reply by KumoLiu** [reply]
Hi @oscarramirezs, since it is a random crop, if you didn't specify the `spatial_size`, it will keep the original size of the label. Thanks.
